src/data/Model.js

```
@@ -187,6 +183,26 @@ class Model {
     }
 
+    /**
+     * Returns a list of properties that are indexed including links to other models.
+     *
+     * @returns {Array<string>} - The indexed properties.
+     * @abstract
+     * @static
+     */
+    static indexedPropertiesResolved() {
+        return [
+            ...Object.entries(this)
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)